crates/forge/benches/my_benchmark.rs

```diff
@@ -0,0 +1,213 @@
+use assert_fs::fixture::PathCopy;
+use assert_fs::TempDir;
+use camino::Utf8PathBuf;
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion, SamplingMode};
+use forge::collecting::{collect_test_compilation_targets, TestCompilationTarget};
+use forge::CrateLocation;
+use indoc::indoc;
+use std::path::PathBuf;
+use std::str::FromStr;
+use std::time::Duration;
+use test_collector::LinkedLibrary;
+use test_utils::corelib::corelib_path;
+use test_utils::runner::{Contract, TestCase};
+use test_utils::running_tests::run_test_case;
+use test_utils::{assert_passed, test_case};
+
+fn setup_collect_tests() -> TempDir {
```

Comment:
Can we split the cases into different files? So that extending/reading this suite is easier?